test/jdk/sun/security/mscapi/VeryLongAlias.java

```diff
@@ -48,7 +48,11 @@
 
     public static void main(String[] args) throws Throwable {
 
-        SecurityTools.keytool("-genkeypair -storetype pkcs12 -keystore ks"
+        // Using the old algorithms to make sure the file is recognized
```

Comment:
Do we also want to have a test that uses the new algorithms?

Reply:
I only know Windows Server 2019 can accept the new algorithms.

Reply:
Ok, but maybe we can split this test in two and use the jtreg @requires tag to run the newer algorithms on Windows Server 2019? It would be a useful test if this is the only test where we test PKCS12 interop with Windows.

Reply:
OK. Or I can see if there is an existing method in test/lib that can detects the version.